lms/migrations/script.py.mako

```diff
@@ -1,23 +1,19 @@
-"""
-${message}
+"""${message}
```

Comment:
The cookiecutter contains the `script.py.mako` exactly as generated by the `alembic ini` in the current version of alembic. Apparently this was different in LMS for some reason. Seems no harm in letting the cookiecutter change this